includes/API/Products.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\Data\API;
+
+use NewfoldLabs\WP\Module\Data\HiiveConnection;
+use WP_REST_Controller;
+use WP_REST_Server;
+
+/**
+ * REST API controller for retriving the products of a customer from the hiive.
+ */
+class Products extends WP_REST_Controller {
+	/**
+	 * Instance of the HiiveConnection class.
+	 *
+	 * @var HiiveConnection
+	 */
+	public $hiive;
```

Comment:
properties should not be `public` unless absolutely necessary. It makes it easier to change things in future when you know that nothing external depends on a a property.